tests/test_eval_api.py

```diff
@@ -0,0 +1,109 @@
+import pytest
```

Comment:
@mkulariya1 create a class for these tests eg.https://github.com/openAGI/tefla/blob/master/tests/test_losses.py#L50

Reply:
done